libs/api/domains/consultation-portal/src/lib/types/locales.enum.ts

```diff
@@ -0,0 +1,4 @@
+export enum Locale {
```

Comment:
Can look at how we use enums with GraphQL elsewhere. E.g.

https://github.com/island-is/island.is/blob/f1aa18299608c925b8ad08e4e803a8bb13b753a5/libs/api/domains/content-search/src/lib/enums/contentLanguage.enum.ts#L8

Reply:
removed since its not used anywhere but will keep it in mind if we need to implement enums later